### DISCUSSION TITLE
Does MMEX use double-entry bookkeeping?

### DISCUSSION THREAD

**Discussion by VictorVow** [moneymanagerex/moneymanagerex#6766] [Q&A]
Can't seem to find the answer to this anywhere.

**Comment by Macaulay1234** [comment]
Yes

**Comment by georgeef** [comment]
Double-entry booking means "Every entry to an account requires a corresponding and opposite entry to a different account" (Wikipedia). Effectively this means all transactions are transfers between accounts (probably including assets).

MMEX has withdrawal and deposit transactions, which are not transfers between accounts. Even in transfer transactions, the amount at the two sides may be different.

So the short answer is: no, MMEX doesn't use double-entry booking.

Conceptually categories can be considered as "accounts" (sink accounts for withdrawals, source accounts for deposits). But this is not what people call "double-entry booking".
